cli/azd/cmd/down.go

```diff
@@ -6,15 +6,40 @@ import (
 
```

Comment:
The diff looks bad since I did a swap between `down.go` and `infra_delete.go` logic.

I did confirm with local diffs that the changes are largely cosmetic. I'm also happy to separate out as a rename PR if that makes it easier for review.

![image](https://user-images.githubusercontent.com/2322434/228090699-582af782-455e-445c-93a3-acead40d6563.png)